### ISSUE TITLE
Issue with "User Profile API" - Wrong schema used in openapi spec

### ISSUE THREAD

**Issue by NJannasch** [PaloAltoNetworks/prisma.pan.dev#211]
## Describe the problem
The REST API Endpoint _/user/me_ returns _MultiRoleUserProfile_ instead of _UserProfileModel_.

In the documentation the expected object has the attribute _roleId_ but _roleIds_ is returned:
https://prisma.pan.dev/api/cloud/cspm/user-profile#operation/get-my-profile

How to reproduce
````
import requests

url = "https://api.prismacloud.io/user/me"
headers = {"x-redlock-auth": "REPLACE_KEY_VALUE"}
response = requests.request("GET", url, headers=headers)
print(response.text)
# Returns: {.... "roleIds": [...] ...}

````

## Suggested fix
Update the returned object in the OpenAPI spec from [_UserProfileModel_ ](https://github.com/PaloAltoNetworks/prisma.pan.dev/blob/d148e2a13f7e0c1b59ec242e11d067dae24d3a6f/static/oas/cspm/UserProfile.yaml#L284)towards _MultiRoleUserProfile_ or make use of _oneOf_, _anyOf_, _allOf_ as described in the openapi specification:
https://swagger.io/docs/specification/data-models/oneof-anyof-allof-not/

**Comment by debbm** [comment]
Thank you for catching this. The fix will be in 22.3.2.

**Comment by debbm** [comment]
We published the fix. Thanks again for pointing out the issue and giving a thoughtful suggested fix.
